tee-launcher/launcher.py

```diff
@@ -43,6 +45,39 @@
 # the unix socket to communicate with Dstack
 DSTACK_UNIX_SOCKET = '/var/run/dstack.sock'
 
+# Example of .user-config file format:
+#
+# MPC_ACCOUNT_ID=mpc-user-123
+# MPC_LOCAL_ADDRESS=127.0.0.1
+# MPC_SECRET_STORE_KEY=secret
+# MPC_CONTRACT_ID=mpc-contract
+# MPC_ENV=testnet
+# MPC_HOME_DIR=/data
+# NEAR_BOOT_NODES=boot1,boot2
+# RUST_BACKTRACE=1
+# RUST_LOG=info
+# MPC_RESPONDER_ID=responder-xyz
+# IMAGE_HASH=sha256:abc123...
+# LATEST_ALLOWED_HASH_FILE=/mnt/shared/image-digest
+# EXTRA_HOSTS=host1:192.168.0.1,host2:192.168.0.2
+# PORTS=11780:11780,2200:2200
+
+# Define an allow-list of permitted environment variables:
+ALLOWED_ENV_VARS = {
```

Comment:
Why didn’t you add `EXTRA_HOSTS` and `PORTS` to `ALLOWED_ENV_VARS`? Did you overlook it?